thirdparty/cmdconfig/commands/cmdsource/cmdsource.go

```diff
@@ -37,15 +35,21 @@ func GetSourceRunner(ctx context.Context, name string) *SourceRunner {
 		Example: fndocs.SourceExamples,
 		Args:    cobra.MaximumNArgs(1),
 		RunE:    r.runE,
+		PreRunE: r.preRunE,
 	}
 	c.Flags().StringVarP(&r.Output, "output", "o", cmdutil.Stdout,
 		fmt.Sprintf("output resources are written to stdout in provided format. Allowed values: %s|%s", cmdutil.Stdout, cmdutil.Unwrap))
 	c.Flags().StringVar(&r.FunctionConfig, "fn-config", "",
 		"path to function config file.")
 	c.Flags().BoolVar(&r.IncludeMetaResources,
 		"include-meta-resources", false, "include package meta resources in the command output")
+	if err := c.Flags().MarkHidden("include-meta-resources"); err != nil {
+		panic(err)
```

Comment:
once this is triggered, users will see the tracing log with kpt source code? Do we want to expose that?